CHANGELOG.md

```diff
@@ -16,6 +16,13 @@
   module's directory.
 - Fix constraints of `--path` flag for lint and breaking rules to avoid resolving all files
   within a module. This change can result in a performance improvement for large workspaces.
+- Add `includes` key to module configurations in v2 `buf.yaml`, accepting a list of directories.
+  * If `includes` is specified, a proto file is considered in the module only if it is in one of the
+    directories specified.
+  * If both `includes` and `excludes` keys are specified for a module, a proto file is considered
+    part of this module if it is contained in any of the include paths and not in any of the exclude
+    paths.
+- Allow multiple module configurations in the same v2 `buf.yaml` to have the same directory path.
```

Comment:
Looks like the CHANGELOG needs to be bumped to the above section.

Reply:
Good catch!